test/e2e/logging_soak.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+)
+
+var _ = framework.KubeDescribe("Logging soak [Experimental] [Slow] [Destructive]", func() {
+
+	f := framework.NewDefaultFramework("logging-soak")
+
+	It(fmt.Sprintf("should survive logging 1KB every %v seconds for %v at scale of %v", .99, 2*time.Minute, framework.TestContext.Scale), func() {
+		defer GinkgoRecover()
+		var wg sync.WaitGroup
+		wg.Add(framework.TestContext.Scale)
```

Comment:
this will allow for the bears attack (2,3,4,5,... pods per node) vs standard (1 pod per node) test